.github/workflows/cli.yml

```diff
@@ -0,0 +1,71 @@
+name: CLI
+
+on:
+  push:
+    branches: [master, 'sdk-*']
+    paths:
+      - .github/workflows/cli.yml
+      - packages/**
+      - yarn.lock
+  pull_request:
+    paths:
+      - .github/workflows/cli.yml
+      - packages/**
+      - yarn.lock
+  schedule:
+    - cron: 0 14 * * *
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event_name }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  test:
+    runs-on: ubuntu-18.04
+    steps:
+      - name: 👀 Checkout a ref for the event
```

Comment:
will need to clean this up later:
1. part of this is a good candidate for an internal composite action
2. only need to check out one commit, not 100